src/arrayMethodSort.js

```diff
@@ -3,9 +3,24 @@
 /**
  * Implement method Sort
  */
+function defaultCompare(a, b) {
+  return String(a) > String(b) ? 1 : -1;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 1; j < this.length - i; j++) {
```

Comment:
The inner loop should start at index 0; starting at 1 misses the first comparison and can lead to incorrect sorting.